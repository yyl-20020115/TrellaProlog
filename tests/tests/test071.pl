:- initialization(main).
:- use_module(library(lists)).

main :-
	findall(I, member(I, [A,B,B,A]), L),
	writeq(L), nl, fail.
main.

