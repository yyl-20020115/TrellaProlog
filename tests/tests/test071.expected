[_7,_8,_9,_10]
